lib/modules/platform/github/index.ts

```diff
@@ -91,6 +91,7 @@ import type {
   PlatformConfig,
 } from './types';
 import { getAppDetails, getUserDetails, getUserEmail } from './user';
+import { normalizeDepName } from '../../datasource/pypi/common';
```

Comment:
please move to utils folder, we don't like to have references from platform to datasource